src/result.rs

```diff
@@ -185,25 +200,6 @@ impl From<Row> for RowWrapper {
     }
 }
 
-/// Converts value into tuple of elements:
-/// ``(typ, value)``
-/// for a given CQL Value.
-///
-/// Intended to be used when there is ambiguity how the given value
-/// should be provided, based on its type.
-///
-/// # Safety
-///
-/// Valid pointer to napi env must be provided
-unsafe fn make_napi_value_and_type_pair(
-    env: napi::sys::napi_env,
-    value: napi::Result<napi::sys::napi_value>,
-    typ: CqlType,
-) -> napi::Result<napi::sys::napi_value> {
-    // Caller of this function ensures a valid pointer to napi env is provided
-    unsafe { Vec::to_napi_value(env, vec![CqlType::to_napi_value(env, typ), value]) }
-}
-
```

Comment:
:tada: I'm so happy this function is finally deleted! It has long confused me and hinted that our approach had a dead end.